pkg/cache/cluster.go

```diff
@@ -702,6 +741,51 @@ func (c *clusterCache) processApi(client dynamic.Interface, api kube.APIResource
 	return nil
 }
 
+// checkPermission runs a self subject access review to check if the controller has permissions to list the resource
+func (c *clusterCache) checkPermission(ctx context.Context, reviewInterface authType1.SelfSubjectAccessReviewInterface, api kube.APIResourceInfo) (keep bool, err error) {
+	sar := &authorizationv1.SelfSubjectAccessReview{
+		Spec: authorizationv1.SelfSubjectAccessReviewSpec{
+			ResourceAttributes: &authorizationv1.ResourceAttributes{
+				Namespace: "*",
+				Verb:      "list", // uses list verb to check for permissions
+				Resource:  api.GroupVersionResource.Resource,
+			},
+		},
+	}
+
+	switch {
+	// if manage whole cluster or resource is cluster level and cluster resources enabled
+	case len(c.namespaces) == 0 || !api.Meta.Namespaced && c.clusterResources:
```

Comment:
```suggestion
	case len(c.namespaces) == 0 || (!api.Meta.Namespaced && c.clusterResources):
```

I know that this is implicit by operator precedence, but imho explicit braces make it much easier to understand the condition

Reply:
sure will update, this condition was taken from an existing function `processApi` so will update it there too.